tree-construction/webkit02.dat

```diff
@@ -551,4 +552,140 @@ eof-in-math
 |             <select>
 |               <option>
 |               <optgroup>
-|               <hr>
+|                 <hr>
+
+#data
+<select><div><i></div><option>option
+#errors
+1:1: ERROR: Expected a doctype token
+1:17: ERROR: End tag 'div' isn't allowed here. Currently open tags: html, body, select, div, i.
+1:37: ERROR: Premature end of file. Currently open tags: html, body, select, option, i.
+#document
+| <html>
+|   <head>
+|   <body>
+|     <select>
+|       <div>
+|         <i>
+|       <option>
+|         <i>
```

Comment:
`<i>` should be the parent per https://github.com/whatwg/html/pull/10557#discussion_r1828992609

Reply:
done